draft-ietf-quic-recovery.md

```diff
@@ -517,9 +520,9 @@ transmit or exceeding the reordering window in time.
 sent_packets:
 : An association of packet numbers to information about them, including a number
   field indicating the packet number, a time field indicating the time a packet
-  was sent, and a bytes field indicating the packet's size.  sent_packets is
-  ordered by packet number, and packets remain in sent_packets until
-  acknowledged or lost.
+  was sent, a boolean indicating whether the packet is ack only, and a bytes
```

Comment:
s/ack only/ackable. I think this constraint needs to be more than just ack only packets.

Reply:
As discussed, in a future PR.